app/src/lib/vbranches/baseBranch.ts

```diff
@@ -20,6 +20,9 @@ import type { Readable } from 'svelte/store';
 
 export class NoDefaultTarget extends Error {}
 
+// Lowercase string of the error message returned from the backend when no target is set.
+const NO_TARGET_SET_ERROR_MESSAGE = 'there is no default target';
```

Comment:
i think you can use error code!

Reply:
It is an `errors.unknown`... let me see if I can add a new code in the rust side

Reply:
That was super simple to introduce!